src/landmarks.rtf

```diff
@@ -0,0 +1,23 @@
+{\rtf1\ansi\ansicpg1252\cocoartf2638
```

Comment:
Is this file still needed? I noticed a spreadsheet that has the same information. Also, is this file incomplete?

Reply:
Not needed.  Deleted